crates/biome_cli/tests/commands/rage.rs

```diff
@@ -83,6 +83,41 @@ fn with_configuration() {
     ));
 }
 
+#[test]
+fn with_jsonc_configuration() {
+    let mut fs = MemoryFileSystem::default();
+    let mut console = BufferConsole::default();
+
+    // issue #2008 is only reproducible when using absolute paths
+    // so we insert an absolute path here
+    fs.insert(
+        PathBuf::from("/dir/biome.jsonc"),
+        r#"{
+  "formatter": {
+    // disable formatter
+    "enabled": false,
+  }
+}"#,
+    );
+
+    // TODO: but how to cd into dir and run rage?
+    let result = run_rage(
+        DynRef::Borrowed(&mut fs),
+        &mut console,
+        Args::from([("rage")].as_slice()),
+    );
```

Comment:
You can use `--config-path` to tell Biome to load the configuration from a different path. In case it should be `--config-path=/dir`.

The problem is that `auto_search` won't kick in, but I suppose it's fine, right? Alt least for the rage command

Reply:
I maybe used an ambiguous name for the test. But my initial intention to add this test is to check whether `auto_search` works correctly or not 😅. Is this achievable?

Reply:
Oh, I see. I think just setting the path `Path::new("biome.jsonc")` is sufficient for the current test case. You also tested it locally, I think it's more than enough :)

Reply:
This is addressed in cdbedb3. Should I be worried about the codspeed report?